sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+- Removed `kind` property on `DocumentPage`.
```

Comment:
Should the changelog also mention that the only supported version going forward in 2022-08-31?

Reply:
We should definitely mention that this version of the library defaults to the GA version